src/App.jsx

```diff
@@ -1,107 +1,29 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId)
+    || null;
+}
+
+function getCommentsById(userId) {
+  return commentsFromServer.find(comments => comments.postId === userId)
+    || null;
+}
```

Comment:
Post id and userId - is defferent value.
My assumption you should use Array.filter() method, because there are a lot of comments with same `postId`
![2023-09-07_12h32_26](https://github.com/mate-academy/react_static-list-of-posts-js/assets/61904995/0080615f-c794-49a1-94a6-0f20cd1df18f)